blocks/article-list/article-list.js

```diff
@@ -1,5 +1,9 @@
 import {
```

Comment:
Ignore formatting changes in article-list.js file. No changes done here. Will revert it later